app/assets/javascripts/components/plant_population_submission.coffee

```diff
@@ -12,6 +12,14 @@ window.PlantPopulationSubmission = class PlantPopulationSubmission extends Submi
     @$('.population-type').select2(@defaultSelectOptions)
     @$('.plant-line-list').select2(@plantLineListSelectOptions)
 
+    fields = [
+      'establishing-organisation'
+    ]
+
+    $.each fields, (_, field) =>
+      @$(".#{field}").select2(@defaultSelectOptions)
+      @$(".#{field}-wrapper").comboField()
```

Comment:
These combo fields do not light red when the validation fails. Could you try to modify their code so they get the 'field_with_errors' css class when the validation fails (all three - the label, the select field and the text input field)? If in doubt about how to do that, please ask @kammerer for assistance.